terraform/gcp/modules/monitoring/infra/alerts.tf

```diff
@@ -246,3 +246,89 @@ resource "google_monitoring_alert_policy" "kms_crypto_request_alert" {
   notification_channels = local.notification_channels
   project               = var.project_id
 }
+
+### Kubernetes Alerts
+
+# Kubernetes Container Memory Utilization > 90%
+resource "google_monitoring_alert_policy" "k8s_container_memory_utilization" {
+  # In the absence of data, incident will auto-close in 7 days
+  alert_strategy {
+    auto_close = "604800s"
+  }
+
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      aggregations {
+        alignment_period   = "300s"
+        per_series_aligner = "ALIGN_MEAN"
+      }
+
+      comparison      = "COMPARISON_GT"
+      duration        = "0s"
```

Comment:
Do we want to increase the duration, say, to 5 minutes? Otherwise quick spikes will cause alerts that aren't actionable.

We should see what the previous duration was and confirm this alert would have fired during the previous outage.

Reply:
I based this on the cloud_sql_memory_utilization alert, this should do a mean aggregation over 5 minutes, and alert if that 5 minute mean is above the threshold. Or am I reading how alerting works incorrectly? Here's that cloudsql alert in gcp if it helps: https://console.cloud.google.com/monitoring/alerting/policies/3138531804338793217/edit?project=project-rekor

Reply:
You're correct, this looks good! Thanks for the link.